squidpy/gr/_ppatterns.py

```diff
@@ -251,20 +234,54 @@ def _moran_helper(
     return pd.DataFrame(moran_list, columns=["I", "pval_sim", "VI_sim"], index=gen)
 
 
-def _compute_moran(y: np.ndarray, w: W, transformation: str, permutations: int) -> Tuple[float, float, float]:
-    mi = esda.moran.Moran(y, w, transformation=transformation, permutations=permutations)
-    return mi.I, mi.p_z_sim, mi.VI_sim
+@njit(
+    ft(ft[:], it[:], it[:], ft[:]),
+    parallel=False,
```

Comment:
In theory, is parallelizable, but I've run into some issues with this recently

Reply:
ok interesting, but didn't see much improvement with parallel anyway